keyboards/handwired/replicazeron/promicro/rules.mk

```diff
@@ -0,0 +1,7 @@
+BOOTMAGIC_ENABLE = no
+CONSOLE_ENABLE = no
+EXTRAKEY_ENABLE = no
+MOUSEKEY_ENABLE = no
+NKRO_ENABLE = no
+RGBLIGHT_ENABLE = no
+LTO_ENABLE = yes
```

Comment:
These can/should be moved to the info.json `qmk migrate` can help with that.

Reply:
done.